examples/gno.land/p/nt/treasury/banker_coins.gno

```diff
@@ -0,0 +1,117 @@
+package treasury
+
+import (
+	"errors"
+	"std"
+
+	"gno.land/p/aeddi/panictoerr"
+)
+
+var ErrNoStdBankerProvided = errors.New("no std banker provided")
+
+// CoinsBanker is a Banker that sends std.Coins.
+type CoinsBanker struct {
+	owner  std.Address // The address of this coins banker owner.
```

Comment:
Shouldn't we specify that this address is intended to be a realm addrs? Maybe it's not the case.

Reply:
There's really no restriction regarding the use through a realm or a user (although I haven't really thought about how a user might use it).